job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/StashPullRequestContext.groovy

```diff
@@ -0,0 +1,69 @@
+package javaposse.jobdsl.dsl.helpers.triggers
+
+import javaposse.jobdsl.dsl.Context
+
+/**
+ * DSL for https://github.com/jenkinsci/stash-pullrequest-builder-plugin
+ */
+class StashPullRequestContext implements Context {
+    String cron = ''
+    String stashHost = ''
+    String username = ''
+    String password = ''
+    String projectCode = ''
+    String repositoryName = ''
+    String ciSkipPhrases = ''
+    boolean checkDestinationCommit = false
+    boolean checkMergeable = false
+    boolean checkNotConflicted = false
+    boolean onlyBuildOnComment = false
+    String ciBuildPhrases = ''
+
+    void cron(String cron) {
+        this.cron = cron
+    }
+
+    void stashHost(String stashHost) {
+        this.stashHost = stashHost
+    }
+
+    void username(String username) {
+        this.username = username
+    }
+
+    void password(String password) {
+        this.password = password
+    }
+
+    void projectCode(String projectCode) {
+        this.projectCode = projectCode
+    }
+
+    void repositoryName(String repositoryName) {
+        this.repositoryName = repositoryName
+    }
+
+    void ciSkipPhrases(String ciSkipPhrases) {
+        this.ciSkipPhrases = ciSkipPhrases
+    }
+
+    void checkDestinationCommit(boolean checkDestinationCommit = false) {
```

Comment:
Change the default parameter value to `true` for all boolean options. It would be confusing if `checkDestinationCommit()` would not check the destination commit.